### PR TITLE
used cs50/common

### PR DESCRIPTION
**Pull request by kzidane**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,76 +1,15 @@
...
-RUN pip install cs50 check50 style50 submit50
+# nltk and punkt data
+RUN nltk && python -m nltk.downloader -d /usr/share/nltk_data/ punkt
 
 # clone checks
 RUN git clone -b master https://github.com/cs50/checks.git /home/ubuntu/.local/share/check50/cs50/checks
 
 # check50 wrapper
-COPY ./check50-wrapper /usr/local/bin/ 
+COPY ./check50-wrapper /usr/local/bin/
```
Comment (dmalan) [inline Dockerfile]: This shouldn't be in a common layer, I assume?

File: Dockerfile [hunk 2]
```diff
@@ -1,76 +1,15 @@
...
-    /opt/pyenv/bin/pyenv global 3.6.0
-ENV PATH "$PYENV_ROOT"/shims:"$PYENV_ROOT"/bin:"$PATH"
-
-# install CS50-specific packages
-RUN pip install cs50 check50 style50 submit50
+# nltk and punkt data
+RUN nltk && python -m nltk.downloader -d /usr/share/nltk_data/ punkt
 
 # clone checks
 RUN git clone -b master https://github.com/cs50/checks.git /home/ubuntu/.local/share/check50/cs50/checks
```
Comment (dmalan) [inline Dockerfile]: How come this is common?

File: Dockerfile (hunk 1 above)
Comment (kzidane) [reply Dockerfile]: It's not.

File: Dockerfile (hunk 2 above)
Comment (kzidane) [reply Dockerfile]: This is ecs :slightly_smiling_face:

**Comment by dmalan** [comment]
Oh I see.

**Pull request merged**